src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,68 @@
+package duke.task;
+import java.time.LocalDate;
```

Comment:
Might be better to leave a space between the package and import statements since you did it for your other files for consistency reasons.